spring-pulsar-dependencies/build.gradle

```diff
@@ -28,5 +28,6 @@ dependencies {
 		api "com.google.protobuf:protobuf-java:$protobufJavaVersion"
 		api "org.apache.pulsar:pulsar-client-all:$pulsarVersion"
 		api "org.apache.pulsar:pulsar-client-reactive-adapter:$pulsarClientReactiveVersion"
+		api "org.apache.pulsar:pulsar-client-reactive-producer-cache-caffeine:$pulsarClientReactiveVersion"
```

Comment:
This should be fine, but just sharing info...

The Reactive adapter brings in caffeine `2.9.3` but we bring in `3.1.1` currently. Gradle version alignment will kick in and pick the latest version - as long as the Reactive adapter is ok w/ that all will be fine. Just a heads up.

Reply:
Unrelated to this PR at this point, but should we shade the caffeine jar for the reactive-client (or is it `implementation` and not on the consumers classpath?)

Reply:
Caffeine jar is `api` in the reactive-client.

Reply:
We might consider shading it at some point.